.github/actions/staging-frontend-deploy/action.yml

```diff
@@ -23,6 +23,17 @@ runs:
   using: "composite"
 
   steps:
+    - name: Set the Slack user
+      shell: python
+      run: |
+        import json
+        import os
+        mapping = json.loads("${{ toJSON(secrets.GH_SLACK_USERNAME_MAP) }}")
+        github_user = "${{ github.actor }}"
```

Comment:
We may want to use `github.triggering_actor` instead[^1], it seems a little more specific in the case of re-runs:

> The username of the user that initiated the workflow run. If the workflow run is a re-run, this value may differ from github.actor. Any workflow re-runs will use the privileges of github.actor, even if the actor initiating the re-run (github.triggering_actor) has different privileges.

[^1]: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
I read those docs, but since the privileges used are the ones of the `github.actor` then it seemed irrelevant to me the use of `github.triggering_actor` 🤔

Anyway! I'll change it in the infra repo too 🙂

Reply:
Done in [6574c02](https://github.com/WordPress/openverse-frontend/pull/2053/commits/6574c0242f822427eb2bc8dd534a4142aae6eed4).